core/modules/user/user.module

```diff
@@ -2793,12 +2800,14 @@ function user_filters() {
   }
 
   $options = array();
+  $module_info = system_get_info('module');
   foreach (module_implements('permission') as $module) {
     $function = $module . '_permission';
+    $module_group = t('@module module', array('@module' => $module_info[$module]['name']));
     if ($permissions = $function()) {
       asort($permissions);
-      foreach ($permissions as $permission => $description) {
-        $options[t('@module module', array('@module' => $module))][$permission] = t($permission);
+      foreach ($permissions as $permission => $permission_info) {
+        $options[$module_group][$permission] = t($permission_info['title']);
```

Comment:
`$permission_info['title']` is already returned translated from `hook_permission()`.

```php
function layout_permission() {
  return array(
    'administer layouts' => array(
      'title' => t('Administer Layouts'),
      'description' => t('Create and modify layouts and blocks.'),
      'restrict access' => TRUE,
      'warning' => t('Change the access/visibility of menu items, layouts and blocks.'),
    ),
    'administer flexible templates' => array(
      'title' => t('Administer Flexible Templates'),
      'description' => t('Create, configure and delete flexible layout templates.'),
      'restrict access' => TRUE,
      'warning' => t('Manage all aspects of flexible layout templates, including deleting them.'),
    ),
  );
}
```

Reply:
Good catch @kiamlaluno 👍🏼 ...fixed.